docs/TESTING.md

```diff
@@ -0,0 +1,25 @@
+# Testing
+
+### E2E tests on BrowserStack
+
+#### Prerequisites
+
+-   [BrowserStack](https://www.browserstack.com) account
+
+### Running tests
+
+This will run all the tests in `showcase/src/e2e-test`
+
+Start the showcase app:
+
+```
+npm run showcase
+```
+
+Start the tests with protractor on BrowserStack, you can find your credentials on [https://automate.browserstack.com/](https://automate.browserstack.com/)
```

Comment:
Can you also explain how to use BrowserStack with Protractor? Maybe you can put a small snippet of the `protractor.conf.js`